microsoft-edge/webview2/reference/javascript/HostObjectsOptions.yml

```
@@ -0,0 +1,116 @@
+      execution of the calling JavaScript and then returning the resolution of the promise, rather than returning a
+      promise.
+
+
+      This defaults to `[]`<!-- -->.
+    remarks: ''
+    isPreview: false
+    isDeprecated: false
+    syntax:
+      content: 'forceAsyncMethodMatches: RegExp[];'
+      return:
+        type: 'RegExp[]'
+  - name: forceLocalProperties
+    uid: 'WebView2Script!HostObjectsOptions#forceLocalProperties:member'
+    package: WebView2Script!
+    fullName: forceLocalProperties
+    summary: >-
+      This is an array of host object property names that will be run locally, instead of being called on the native
+      host object. This defaults to `['then', 'toJSON', Symbol.toString, Symbol.toPrimitive]`<!-- -->. You can add other
+      properties to specify that they should be run locally on the javascript host object proxy.
+    remarks: ''
+    isPreview: false
+    isDeprecated: false
+    syntax:
+      content: 'forceLocalProperties: string[];'
+      return:
+        type: 'string[]'
+  - name: ignoreMemberNotFoundError
+    uid: 'WebView2Script!HostObjectsOptions#ignoreMemberNotFoundError:member'
+    package: WebView2Script!
+    fullName: ignoreMemberNotFoundError
+    summary: >-
+      By default, an exception is thrown when attempting to get the value of a proxy property that doesn't exist on the
+      corresponding native class. Setting this property to `true` changes the behavior to match Chakra WinRT projection
+      (and general JavaScript) behavior of returning `undefined` with no error.
+    remarks: ''
+    isPreview: false
+    isDeprecated: false
+    syntax:
+      content: 'ignoreMemberNotFoundError: boolean;'
+      return:
+        type: boolean
+  - name: log
+    uid: 'WebView2Script!HostObjectsOptions#log:member'
+    package: WebView2Script!
+    fullName: log
+    summary: >-
+      This is a callback delegate that will be called with debug information if non-null.
+
+
+      For example, you can set this to `console.log.bind(console)` to have it print debug information to the console to
+      help when troubleshooting host object usage.
+
+
+      By default this is `null`<!-- -->.
+    remarks: ''
+    isPreview: false
+    isDeprecated: false
+    syntax:
+      content: 'log: (...data: any[]) => void;'
+      return:
+        type: '(...data: any[]) =&gt; void'
+  - name: shouldSerializeDates
+    uid: 'WebView2Script!HostObjectsOptions#shouldSerializeDates:member'
+    package: WebView2Script!
+    fullName: shouldSerializeDates
+    summary: >-
+      By default this is `false`<!-- -->, and javascript `Date` objects will be sent to host objects as a string using
+      `JSON.stringify`<!-- -->.
+
+
+      You can set this property to `true` to have `Date` objects properly serialize as a `VT_DATE` when sending to the
+      native host object, and have `VT_DATE` properties and return values create a JavaScript `Date` object.

```

```suggestion
      native host object, and to have `VT_DATE` properties and return values create a JavaScript `Date` object.
```
tech review this change

---

Adding the `to` changes the meaning of the sentence. Marking this as "won't fix".